content/administration/upgrade.rst

```diff
@@ -1,264 +1,340 @@
-      team. This will slow down the processing and response time.
+         $ python <(curl -s https://upgrade.odoo.com/upgrade) --help
 
-Please provide as much detail as you can (i.e., videos and screenshots to illustrate your issue).
-This will avoid clarifying questions and speed up the resolution process significantly.
+      An upgraded test database can also be requested via the `Upgrade page
+      <https://upgrade.odoo.com/>`_.
+
+      .. note::
+         - For security reasons, only the person who submitted the upgrade request can download it.
+         - For storage reasons, the database's copy is submitted without a filestore to the upgrade
+           server. Therefore, the upgraded database does not contain the production filestore.
+         - Before restoring the upgraded database, its filestore must be merged with the production
+           filestore to be able to perform tests in the same conditions as it would be in the new
+           version.
+         - The upgraded database contains:
+
+           - A `dump.sql` file containing the upgraded database
+           - A `filestore` folder containing files extracted from in-database records into
+             attachments (if there are any) and new standard Odoo files from the targeted Odoo
+             version (e.g., new images, icons, payment provider's logos, etc.). This folder be
+             merged with the production filestore to get the full upgraded filestore.
 
 .. note::
-   * The purpose of the test phase is not to correct existing data or configurations in your
-     database.
-   * |assistance-contact|
+   You can request multiple test databases if you wish to test an upgrade more than once.
 
-.. _upgrade/steps-production:
+In all cases, the process is the same:
 
-The production launch
-=====================
+#. Exporting the database to a file
+#. Uploading the file to the upgrade server
+#. Running a series of :ref:`standard migration scripts <reference/upgrade/migration-scripts>` to
+   upgrade standard modules
+#. Downloading the upgraded database
+#. Importing the file into a database
+#. (*Optional*) Running a series of :ref:`custom migration scripts
+   <reference/upgrade/migration-scripts>` developed by third parties to upgrade custom modules
 
-The production upgrade request is when you decide to upgrade your current database with all your
-production data (invoices, VAT returns, inventories, current orders) to a new version of your
-choice.
+.. _upgrade/upgrade_report:
 
-After your :ref:`tests <upgrade/testing-phase>` are completed to your satisfaction, submit the
-request to upgrade your production database via our `website form <https://upgrade.odoo.com>`_.
-Select *Production* purpose.
+.. note::
+   When an upgrade request is completed, an upgrade report is emailed and made available in the
+   Discuss app. It contains important information about the changes introduced by the new version.
```

Comment:
```suggestion
   When an upgrade request is completed, an upgrade report is appended to the successful upgrade
   email and it becomes available in the Discuss app for users who are part of the "Administration
   / Settings" group. This report provides important information about the changes introduced by the
   new version.
```

Reply:
I agree but "attached" and not "appended" would sound better IMO, adding it as my own commit to do the change